app/tests/api/CategoriesTestCest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class CategoriesTestCest
+{
+    public function RetrieveAllCategoriesViaAPI(ApiTester $I)
+    {
+        $I->haveHttpHeader('Content-Type', 'application/json');
+        $I->haveHttpHeader('Authorization', 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9leGFtcGxlLm9yZyIsInVzZXIiOnsiaWQiOiIxIiwidXNlcl9uYW1lIjoiamVmZmkifSwiaWF0IjoxNjA1MzA1NzUxfQ.71WUCmFy0RoIFv8zryFMLs_zuSTp0-1VQdbXvnUIq-s');
```

Comment:
Put this Authorization token on a login function, to avoid duplicate the same set bearer thing.